src/lib/util/domains/domains.test.ts

```diff
@@ -0,0 +1,56 @@
+//- Utils Imports
+import { getDomainId, truncateAddress } from './domains';
+
+//- Mocks Imports
+import { rootDomainId, wilderDomainId } from './mocks';
+
+//////////////////////
+// f:: getDomainId  //
+//////////////////////
+
+describe('getDomainId', () => {
+	describe('when args is equal to empty string', () => {
+		it('should return the rootDomainId (hashZero)', () => {
+			expect(getDomainId('')).toBe(rootDomainId);
+		});
+	});
+
+	describe('when args is equal to undefined', () => {
+		it('should return the rootDomainId (hashZero)', () => {
+			expect(getDomainId(undefined)).toBe(rootDomainId);
+		});
+	});
```

Comment:
Is this the behaviour that we want? I think it would be best if the parameter isn't optional. If it needs to be optional, I wouldn't expect `undefined` to return `rootDomainId` - I would probably expect it to return `undefined`.